apps/interpreter/src/run-options.ts

```diff
@@ -0,0 +1,181 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
```

Comment:
Since you've moved things to a class, does it make sense to continue doing that with the run options as well? They all belong together.

Also, it might make sense to move `parsePipelineMatcherRegExp` there.

Reply:
Do you mean turning the `RunOptions` from an interface to a class?

Reply:
Rather move all of these helper functions to a class

Reply:
In this specific case, I don't see the big benefit compare to a collection of methods. We can refactor it when we really need it.

For the interpreter, I saw a larger benefit as we had methods with 5+ parameters. It should also pave the way to configure the workspace directory more easily.

If you don't insist on that change, I'd skip it 😅

Reply:
sure lets skip it :+1: